.github: Notify teams as part of filing a CFP (#39298)
cilium-team/cilium: .github/ISSUE_TEMPLATE/feature_template.md

Include any specific requirements you need

**Notify relevant community channels**

> Consider raising this topic on [Slack] in the #development channel or during a

Reviewer: Might want to give some instructions if the sig is not obvious.

Author: Do you have an idea what that might look like?

Ideas:
- Include any area you think might be relevant, the recipients can decide which area it's more related to
- Use the CODEOWNERS file to find which files you might change and locate teams that way

Reviewer: I think going broad and then having the sigs understand that they need unmark themselves would be a good start. Pinging on development as well is ok -- but I've found the response rate might not be predictable?

Reviewer 2: I think "response rate unpredictability" is just how it is in open source - as a group of volunteers, nobody is under any obligation to respond. But the broad notification to the development channel seems like it's the best place to find other folks who are interested, and the wider the interest in an idea, the more likely that SIG folks and committers will want to help

Reviewer 3: I've switched around the wording to address Bowie's comment if the SIG is not clear

Author: I accepted Bill's wording and also amended to simplify the language:

> If you don't know which team to include or you haven't gotten feedback, consider raising this topic on [Slack] in the #development channel or during a [community meeting] for awareness.

> [community meeting] for awareness. List any relevant SIGs below from the
> [teams] list in the following form so that members of those teams are notified:

Reviewer 2: Not obvious to me what "List" means here, but is this right? 

```suggestion
[community meeting] for awareness. Notify the members of any relevant SIGs from the
[teams] list by mentioning them in the issue in the following format:
```

Author: Applied along with Bill's suggestion, thanks.

> Consider raising this topic on [Slack] in the #development channel or during a
> [community meeting] for awareness. List any relevant SIGs below from the
> [teams] list in the following form so that members of those teams are notified:
> 
> - @cilium/example

Reviewer 3: ```suggestion
List any relevant SIGs below from the [teams] list in the following form so that members of those teams are notified:

- @cilium/example

If the SIG is not clear or you haven't gotten feedback, consider raising this topic on [Slack] in the #development channel or during a [community meeting] for awareness. 
```
